test/Microsoft.AspNetCore.SignalR.Test.Server/Startup.cs

```diff
@@ -9,7 +9,8 @@ public class Startup
     {
         public void ConfigureServices(IServiceCollection services)
         {
-            services.AddRouting();
```

Comment:
Ah okay. Looks like it was added to AddSockets here https://github.com/aspnet/SignalR/commit/1b59fc6f804e4576e704e6c6f8f9aa90b79ab64d